Monika After Story/game/script-topics.rpy

```diff
@@ -1146,10 +1146,11 @@ label monika_simulated:
 
 
 init 5 python:
-    for key in ['rain','sound','weather']:
-        monika_topics.setdefault(key,[])
-        monika_topics[key].append('monika_rain')
-    monika_random_topics.append('monika_rain')
+    if not seen_event('monika_rain'):
```

Comment:
Why are you adding this check? In terms of random chatter, there are already checks in place to prevent topics from being seen multiple times. In terms of the keyword system, it is **intended** to allow topics to be repeated.

Reply:
I suggested this check. It will no longer make sense for monika to talk about wanting to add something that's already added, even after all topics have been seen or the keyword is seen it would be odd to see Monika talk about wanting to add something that is already there.

Reply:
Okay, I get what you're saying. However, the `rain` topic doesn't seem to suggest that she's wants to add rain, just that she likes rain.